diagnostic.md

```diff
@@ -94,4 +92,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+The value of `teamMember['hobby']` is the string 'exercise'. The value of `teamMember['name']` is the array `['given', 'surname']`. The value of `teamMember['name']['surname']` is the string 'Weeks'.
```

Comment:
2 out of 3

for `teamMember['name']` we have an object that has 2 keys, 'given' and 'surname' which have values so the correct return value will be { given: 'Jason', surname: 'Weeks' };